### ISSUE TITLE
[Bug]: + button not accessable for blind users

### ISSUE THREAD

**Issue by ruudux** [nextcloud/server#33377]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The + button in the webinterface of de file browser in Nextcloud to create folders etc, can't be find using screenreaders like the opensource nvda or Dolphin screenreader. A blind admin can't create folders or shares. The button doesn't have a alttag on mouseover.

### Steps to reproduce

1.open the webinterface and open files
2.try to find the + button after the home to create or share a folder using a screenreader like nvaccess.org
3.

### Expected behavior

Would be nice when the button is tabable, marked with an altag and accessable/findable for a blind user

### Installation method

Web installer on a VPS or web space

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
not relevant
```

### List of activated Apps

```shell
not relevant
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
not relevant
```

### Additional info

not relevant

**Comment by PVince81** [comment]
@ruudux this should be fixed already on master / the upcoming Nextcloud 25

are you able to try that version ?

**Comment by ruudux** [comment]
Sorry, I hat to update this issue. The button isn't accessable when there isn't a directorie/share. I created a folder using the nextcloud app and since that moment, the webinterface gives me the new button in de file browser. Slidly different the a not accessable button, the button is only not seen when there is'nt a folder. Will update my instance as soon as possible

**Comment by PVince81** [comment]
you should update to at least Nextcloud 24 if you're on an older version
the upcoming Nextcloud 25 (master) will have many accessibilitiy improvements

it seems the issue you see might be unrelated if it's about button visibility.
can you post a screenshot from before/after so we can understand what page you are talking about ?

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.
